react/modules/sample/src/components/BoundaryFilter.js

```
@@ -0,0 +1,773 @@
+
+  //for unselect logic 
+  const cleanLowerLevels = (boundaryType, codesToRemove, updatedOptions) => {
+    if (codesToRemove.length == 0) return updatedOptions;
+    let newBoundaryOptions = recursiveCleanup(boundaryType, updatedOptions, codesToRemove);
+    return newBoundaryOptions; // Return the final updated state
+  };
+
+  //for unselect logic 
+  const cleanLowerLevelsForSelectedValues = (boundaryType, removedCodes, updatedOptions) => {
+    debugger;
+    return updatedOptions.filter(option => {
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)